inc/targetinterface.class.php

```diff
@@ -37,5 +37,27 @@ interface PluginFormcreatorTargetInterface
 {
    public function save(PluginFormcreatorFormAnswer $formanswer);
 
-   public function getTargetItemtypeName();
+   public static function getTargetItemtypeName();
+
+   /**
+    * Tell if this target does an action and does not generate an object
+    *
+    * @return boolean true if the target does an action
+    */
+   public function isActionTarget(): bool;
```

Comment:
You set multiple similar functions as static in this PR, maybe this one should be static too ?

Reply:
I agree, but not in this PR. And I focus on releasing an Alpha 4. I wanted to release it last week already.